models/lido_liquidity/kyberswap_ethereum/lido_liquidity_kyberswap_ethereum.sql

```diff
@@ -0,0 +1,270 @@
+{{ config(
+    alias = 'lido_liquidity_kyberswap_ethereum',
+    partition_by = ['time'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['pool', 'blockchain', 'time'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "lido",
+                                \'["ppclunghe", "gregshestakovlido"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2022-10-28' %} 
+
+with dates as (
+select explode(sequence(to_date('{{ project_start_date }}'), now(), interval 1 hour)) as hour
+)
+ 
+, pools as (
+    select pool as address, 'Ethereum' as blockchain, 'Kyberswap' as project, swapFeeUnits/1000 as fee
```

Comment:
minor, but we typically keep blockchain & project text values lowercase across spellbook. if this is a one-off spell and you feel strongly to keep as-is, that's fine, but just a note